lib/rules/at-rule-property-required-list/README.md

```diff
@@ -1,6 +1,6 @@
 # at-rule-property-required-list
 
-Specify a list of required descriptors (previously labelled as properties) for an at-rule.
+Specify a list of required properties (or descriptors) for an at-rule.
```

Comment:
@jeddy3 we forgot to update

https://github.com/stylelint/stylelint/blob/8ef1792d08755282b74216ead1508af94556538d/docs/user-guide/rules.md?plain=1#L172

Reply:
Thanks for the catch. Will fix.